### PR TITLE
refactor(submodel repository): create separate env variable for submodel repository

### PR DESCRIPTION
**Pull request by Xmilofranke**
# Description

The change enables users to configure a separate aas repository and submodel repository

Fixes # (Issue)

## Type of change

Please delete options that are not relevant.

-   [x] Minor change (non-breaking change, e.g. documentation adaption)
-   [ ] Bug fix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that causes existing functionality to not work as expected)

# Checklist:

-   [x] I have performed a self-review of my code
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [x] I have made corresponding changes to the documentation
-   [x] My changes generate no new warnings
-   [ ] I have added tests that prove my fix or my feature works
-   [ ] New and existing tests pass locally with my changes
-   [x] My changes contain no console logs

### CODE REVIEW

File: .env.local [hunk 1]
```diff
@@ -3,9 +3,11 @@ LOCK_TIMESERIES_PERIOD_FEATURE_FLAG: true
 COMPARISON_FEATURE_FLAG: true
 AAS_LIST_FEATURE_FLAG: true
 AAS_REPO_API_URL: 'http://localhost:5064/repo'
+SUBMODEL_REPO_API_URL: 'http://localhost:5064/repo'
 MNESTIX_BACKEND_API_URL: 'http://localhost:5064'
 DISCOVERY_API_URL: 'http://localhost:5064/discovery'
 REGISTRY_API_URL: 'http://localhost:8083'
+REGISTRY_API_URL: 'http://localhost:8083'
```
Comment (GailMelanie) [inline .env.local:10]: I guess this duplication was accidental?

File: .env.local [hunk 2]
```diff
@@ -3,9 +3,11 @@ LOCK_TIMESERIES_PERIOD_FEATURE_FLAG: true
 COMPARISON_FEATURE_FLAG: true
 AAS_LIST_FEATURE_FLAG: true
 AAS_REPO_API_URL: 'http://localhost:5064/repo'
+SUBMODEL_REPO_API_URL: 'http://localhost:5064/repo'
```
Comment (GailMelanie) [inline .env.local:6]: Do you plan to add this variable to the documentation in another PR?

Comment (Xmilofranke) [reply .env.local:6]: documentation is now updated

File: .env.local (hunk 1 above)
Comment (Xmilofranke) [reply .env.local]: oops, fixed now

**Review by GailMelanie — APPROVED** [review]

**Review by NilsXitaso — APPROVED** [review]

**Pull request merged**